protocol/avdl/stellar1/local.avdl

```diff
@@ -327,6 +327,9 @@ protocol local {
   KeybaseRequestID makeRequestLocal(int sessionID, string recipient, union { null, Asset } asset, 
     union { null, OutsideCurrencyCode } currency, string amount, string note);
 
+  void setAccountMobileOnlyLocal(int sessionID, AccountID accountID);
```

Comment:
take bool to unset?

Reply:
don't want to expose unset quite yet, as only mobile devices would be able to unset and it might be more complicated than an RPC call...

Reply:
Not sure why it would be more complicated. It might be annoying to add later since the argument `enabled bool` would default to false. But if you want ok